lib/node_modules/@stdlib/stats/iter/README.md

```diff
@@ -128,91 +174,91 @@ console.log( getKeys( ns ) );
 
 <!-- <toc-links> -->
 
-[@stdlib/stats/iter/cugmean]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/iter/cugmean
+[@stdlib/stats/iter/cugmean]: https://github.com/stdlib-js/stats/tree/main/iter/cugmean
```

Comment:
Why did you change these URLs? These changes need to be reverted. All URLs within the main project repo should refer to resources in the main project repo, not standalone repos.